monsieurkayode/Play-Nine
# #60827919 Setup development environment

#### What does this PR do?
Add project configurations for building the application
#### Description of Task to be completed?
- Install dependencies and tools
#### How should this be manually tested?
- Clone the repo and checkout to branch `ch-setup-development-tools-160827919`
- Run `yarn` in your terminal to install dependencies
- Run `yarn dev` to start the application
- Navigate to [http://localhost:8080](http://localhost:8080)
- You should see `Play Nine` displayed on the page
#### What are the relevant pivotal tracker stories?
- [#60827919](https://www.pivotaltracker.com/story/show/160827919)
#### Background Context
None
